extension/js/common/composer.ts

```diff
@@ -1570,6 +1570,7 @@ export class Composer {
     this.S.cached('compose_table').css('display', 'none');
     this.S.cached('reply_msg_successful').find('div.replied_from').text(this.getSender());
     this.S.cached('reply_msg_successful').find('div.replied_to span').text(msg.headers.To.replace(/,/g, ', '));
+    plaintext = this.formatTextForReply(plaintext);
```

Comment:
You can use `PgpMsg.stripFcTeplyToken` instead.

Reply:
oh, I didn't know there is that function